core/lib/zksync_core/src/api_server/web3/state.rs

```diff
@@ -594,7 +594,7 @@ impl Filters {
     /// Instantiates `Filters` with given max capacity.
     pub fn new(max_cap: usize) -> Self {
         Self {
-            state: Default::default(),
+            state: LruCache::unbounded(),
```

Comment:
true, but I had problems with making it bounded and spent a lot of time of trying to fix that😀 generally,I had a problem with hashbrown dependency in lru crate, which should be disabled by 'default-features = false', but that didn't help, so that was the only thing that worked